robotino_bringup/robots/uibk-robotino2-sh/launch/uibk-robotino2-sh.launch

```diff
@@ -8,7 +8,8 @@
 
   <!-- COMMON -->
   <!-- send robot urdf to param server -->
-  <param name="robot_description" command="$(find xacro)/xacro.py '$(find robotino_bringup)/robots/$(arg robot)/urdf/robotino.urdf.xacro'" />
```

Comment:
as mentioned in the mail, using the robotino.urdf.xacro results in missing links (bsaelinkx,-y,-z); using the generated urdf file (from the robotino_base_plan.urdf.xacro) results all links being there.